packages/data-stores/src/reader/mutations/use-site-unfollow-mutation.ts

```diff
@@ -0,0 +1,51 @@
+import { useMutation, useQueryClient } from 'react-query';
+import { callApi } from '../helpers';
+import { useIsLoggedIn } from '../hooks';
+
+type SiteSubscriptionUnfollowParams = {
+	blog_id: number | string;
+};
+
+type SiteSubscriptionUnfollowResponse = {
+	success: boolean;
+	subscribed: boolean;
+	subscription: null;
```

Comment:
Is this correct? Is this expected to be `null`? 🤔

Reply:
Yes, it is:

<img width="948" alt="CleanShot 2023-04-25 at 13 50 15@2x" src="https://user-images.githubusercontent.com/528287/234267722-aa17dc4d-2b89-4324-a042-5a17b7fa4779.png">